subsys/bluetooth/shell/bt.c

```diff
@@ -659,7 +659,8 @@ static int cmd_advertise(const struct shell *shell, size_t argc, char *argv[])
 {
 	struct bt_le_adv_param param;
 	const struct bt_data *ad, *scan_rsp;
-	size_t ad_len, scan_rsp_len;
+	size_t ad_len;
+	size_t scan_rsp_len = 0;
```

Comment:
Actually, I don't see scan_rsp_len and scan_rsp used for anything in this function, so you can just remove them from this function and pass directly NULL and 0 to bt_le_adv_start();

Reply:
That makes sense!